docs/modules/deployment/pages/core/ubuntu/install-core.adoc

```diff
@@ -1,13 +1,14 @@
 .Add OpenNMS repository GPG key
 [source, console]
 ----
-sudo apt-key adv --fetch-keys https://debian.opennms.org/OPENNMS-GPG-KEY
+curl -fsSL https://debian.opennms.org/OPENNMS-GPG-KEY | sudo gpg --dearmor -o /usr/share/keyrings/opennms.gpg
 ----
 
-.Add apt repository
+.Add and update apt repository
 [source, console]
 ----
-sudo add-apt-repository -s 'deb https://debian.opennms.org stable main'
+echo "deb [signed-by=/usr/share/keyrings/opennms.gpg] https://debian.opennms.org stable main" | sudo tee /etc/apt/sources.list.d/opennms.list
```

Comment:
Just FYI: add-apt-repository can't handle the signed-by directive see https://bugs.launchpad.net/ubuntu/+source/software-properties/+bug/1862764